package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "test-front-end-taqtile",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.3.1",
```

Normalmente setamos uma versão especifica das libs para evitar problemas com breaking changes, consegue fazer isso com:
```suggestion
    "react": "18.3.1",
```

Ai basta replicar para todas as libs instaladas, e depois dar um npm install novamente para mudar seu package-look